cardano-db-sync-extended/app/cardano-db-sync-extended.hs

```diff
@@ -28,7 +36,22 @@ main = do
   cmd <- Opt.execParser opts
   case cmd of
     CmdVersion -> runVersionCommand
-    CmdRun params -> runDbSyncNode extendedDbSyncNodePlugin params
+    CmdRun params -> do
+
+        -- Configure logging first, top-level dependency
+        let configFile = enpConfigFile params
+        enc <- readDbSyncNodeConfig configFile
```

Comment:
This should all be in `cardano-db-sync` so that both the regular and the extended versions can use it.

Reply:
Fixed.